include/zephyr/bluetooth/audio/lc3.h

```diff
@@ -254,38 +254,50 @@ enum bt_audio_codec_config_type {
+		BT_AUDIO_CODEC_CAP_DATA(BT_AUDIO_CODEC_LC3_FRAME_LEN,                              \
+					BT_BYTES_LIST_LE16(_len_min),                              \
+					BT_BYTES_LIST_LE16(_len_max)),                             \
 		COND_CODE_1(_max_frames_per_sdu, (),                                               \
-			    (, BT_AUDIO_CODEC_DATA(BT_AUDIO_CODEC_LC3_FRAME_COUNT,                 \
-								   _max_frames_per_sdu)))          \
+			    (BT_AUDIO_CODEC_CAP_DATA(BT_AUDIO_CODEC_LC3_FRAME_COUNT,               \
+						     (_max_frames_per_sdu)))),                     \
 	}
 
 /**
  *  @brief Helper to declare LC3 codec metadata
  */
-#define BT_AUDIO_CODEC_LC3_META(_prefer_context)                                                   \
+#define BT_AUDIO_CODEC_CAP_LC3_META(_prefer_context)                                               \
 	{                                                                                          \
-		BT_AUDIO_CODEC_DATA(BT_AUDIO_METADATA_TYPE_PREF_CONTEXT,                           \
-				    BT_BYTES_LIST_LE16(_prefer_context))                           \
+		BT_AUDIO_CODEC_CAP_DATA(BT_AUDIO_METADATA_TYPE_PREF_CONTEXT,                       \
+					BT_BYTES_LIST_LE16(_prefer_context))                       \
 	}
 
 /**
- *  @brief Helper to declare LC3 codec
+ * @brief Helper to declare LC3 codec
+ *
+ * @param _cid Company ID
+ * @param _vid Vendor ID
+ * @param _freq Supported Sampling Frequencies bitfield (see BT_AUDIO_CODEC_LC3_FREQ_*)
+ * @param _duration Supported Frame Durations bitfield (see BT_AUDIO_CODEC_LC3_DURATION_*)
+ * @param _chan_count Supported channels (see @ref BT_AUDIO_CODEC_LC3_CHAN_COUNT_SUPPORT)
+ * @param _len_min Minimum number of octets supported per codec frame
+ * @param _len_max Maximum number of octets supported per codec frame
+ * @param _max_frames_per_sdu Supported maximum codec frames per SDU
+ * @param _prefer_context Preferred contexts (@ref bt_audio_context)
+ *
  */
-#define BT_AUDIO_CODEC_LC3(_freq, _duration, _chan_count, _len_min, _len_max, _max_frames_per_sdu, \
-			   _prefer_context)                                                        \
-	BT_AUDIO_CODEC(BT_AUDIO_CODEC_LC3_ID, 0x0000, 0x0000,                                      \
-		       BT_AUDIO_CODEC_LC3_DATA(_freq, _duration, _chan_count, _len_min, _len_max,  \
-					       _max_frames_per_sdu),                               \
-		       BT_AUDIO_CODEC_LC3_META(_prefer_context))
+#define BT_AUDIO_CODEC_CAP_LC3(_cid, _vid, _freq, _duration, _chan_count, _len_min, _len_max,      \
+			       _max_frames_per_sdu, _prefer_context)                               \
+	BT_AUDIO_CODEC_CAP(BT_AUDIO_CODEC_LC3_ID, _cid, _vid,                                      \
+			   BT_AUDIO_CODEC_CAP_LC3_DATA(_freq, _duration, _chan_count, _len_min,    \
+						       _len_max, _max_frames_per_sdu),             \
+			   BT_AUDIO_CODEC_CAP_LC3_META(_prefer_context))
```

Comment:
```suggestion
#define BT_AUDIO_CODEC_CAP_LC3(_freq, _duration, _chan_count, _len_min, _len_max,      \
			       _max_frames_per_sdu, _prefer_context)                               \
	BT_AUDIO_CODEC_CAP(BT_AUDIO_CODEC_LC3_ID, 0x0000, 0x0000,                                      \
			   BT_AUDIO_CODEC_CAP_LC3_DATA(_freq, _duration, _chan_count, _len_min,    \
						       _len_max, _max_frames_per_sdu),             \
			   BT_AUDIO_CODEC_CAP_LC3_META(_prefer_context))
```

As per PACS v1.0
 
![image](https://github.com/zephyrproject-rtos/zephyr/assets/10126877/7cc37e06-c908-49fa-a772-175e1fa52d9c)

Meaning that Vendor codecs shall set Coding_Format to 0xFF. Other Coding_Format's are reserved for non-vendor Codec ID's, so CID and VID shall be 0x0000 in this case

Reply:
Nice catch!